testsuite/integration/docker-compose.yml

```diff
@@ -0,0 +1,87 @@
+version: "2.2"
+
+services:
+  mysql:
+    image: mysql:5.7
```

Comment:
5.7 doesn't work on my M1 mac, but does work on my intel mac.  I switched this to mysql:8 and it worked on my M1.